### DISCUSSION TITLE
40k+ NDCs in Athena that I can't find elsewhere

### DISCUSSION THREAD

**Discussion by jrlegrand** [OHDSI/Vocabulary-v5.0#911] [Q&A] [answered]
Hello - I've been aggregating NDCs from RxNorm (which includes MTHSPL - DailyMed SPL NDCs), RxNorm Historical, FDA NDC Directory, FDA Excluded, and FDA Unfinished.  Across all those sources, I can't find matches for these 40k+ NDCs.  Can you help me figure out where these NDCs are coming from?  They appear to mostly be OTC / maybe homeopathic / maybe DME or non-meds.  Are they from archived DailyMed SPLs?  Or elsewhere?

Any help is appreciated - thank you!

See attached file below for the NDCs in question.

[mystery_ndcs.xlsx](https://github.com/OHDSI/Vocabulary-v5.0/files/13313353/mystery_ndcs.xlsx)

**Comment by cgreich** [comment, marked as answer]
@jrlegrand:

You don't actually have to do that. The OHDSI vocabulary has already consolidated these mappings to RxNorm in one place, and apparently from places you haven't tried, since some of the products on the list are mapped (e.g. 47234031569 "Iopamidol 610 MG/ML Injectable Solution [Isovue-300]"). 

But a lot of the NDCs are missing from any drug repository. Reason: There are OTC which companies throw on the market in rapid pace and either are rather undefined (e.g. 00440854090 "THERAPEUTIC VIT-MINERAL TAB"), contain ingredients that do not need FDA approval (e.g. 49452154002 "CALCIUM-D-PANTOTHENATE PWDR"), or are no drugs at all (e.g. 11822316520 "RA CUSHION INSOLE MENS").

But we are not completely perfect in our mappings, either. It's just a never ending chase. But the drugs that are worth studying in large scale observational studies should be there. If you are into vitamin combinations - not so good.
